assets/docs/neso613/ASR_TFLite/pre_trained_models/English/1.md

```diff
@@ -0,0 +1,41 @@
+# Lite neso613/ASR_TFLite/pre_trained_models/English/1
+
+TF Lite quantized version of the Conformer to generate text from audio.
+
+<!-- parent-model: neso613/ASR_TFLite/pre_trained_models/1 -->
+<!-- asset-path: legacy -->
+
+ASR TFLite model is 19.1 MB in size[link.](https://drive.google.com/file/d/1jElSz35ivHsW7ww5X85rghEf0PVDlaMZ/view?usp=sharing)
```

Comment:
No need to include this. The model size shows up on the download button.

Reply:
ok. I will remove this. Thanks for feedback.